.travis.yml

```diff
@@ -1,13 +1,33 @@
 sudo: false
+
 language: python
+
 cache: pip 
+
 python:
   - 3.5
+
+env:
+  - AMY_ENABLE_PYDATA=true
+    AMY_PYDATA_USERNAME=username
+    AMY_PYDATA_PASSWORD=password
+  - CHECK_MIGRATION=true
```

Comment:
1) Is this necessary? We don't have two Travis jobs anymore.
2) Previously, you couldn't enable pydata and, at the same time, enable migrations checks. What did you changed, that it works now?